src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/RightCurly/context/defaultContextConfig.xml

```diff
@@ -9,7 +9,7 @@
         <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionJavaPatchFilter">
             <property name="file" value="${tp}/defaultContext.patch" />
             <property name="strategy" value="context" />
-            <property name="neverSuppressedChecks" value="RightCurlyCheck" />
+            <property name="checkNamesForContextStrategyByTokenOrAncestorSet" value="RightCurlyCheck" />
```

Comment:
No difference with neverSuppressedChecks results. We should add violation which will be suppressed after this update. Please add another try-catch (in the same method in Test.java), which produces violation, but will be suppressed by patch-filter.